src/broker/broker.c

```diff
@@ -766,7 +767,8 @@ static const char *cas_client_type_str[] = {
   "ODBC",			/* CAS_CLIENT_ODBC */
   "JDBC",			/* CAS_CLIENT_JDBC */
   "PHP",			/* CAS_CLIENT_PHP */
-  "OLEDB"			/* CAS_CLIENT_OLEDB */
+  "OLEDB",			/* CAS_CLIENT_OLEDB */
+  "INTERNAL_JDBC"		/* CAS_CLIENT_SERVER_SIDE_JDBC */
```

Comment:
'INTERNAL_JDBC' is not used.

Reply:
cas_calient_type_str used in receiver_thr_f () and it is accessed with an index declared at CAS_CLIENT_TYPE enumeration type.